src/sdk/pynni/nni/bohb_advisor/bohb_advisor.py

```diff
@@ -573,3 +573,30 @@ def handle_report_metric_data(self, data):
 
     def handle_add_customized_trial(self, data):
         pass
+
+    def handle_feed_tuning_data(self, data):
```

Comment:
I don't think you need a new function here, you just need to use receive_trial_result. Otherwise you just create a "load_checkpoint" function....

Reply:
This is the design result of the discussion at the previous meeting.

Reply:
This is the implementation of the abstract function of the based class (msg_dispatcher_base). There might be other tuners that may need a new function to implement it so the abstract function prototype is a general one.